src/com/lilithsthrone/controller/MainControllerInitMethod.java

```diff
@@ -1405,6 +1405,99 @@ public void effects() {
 			if(Main.game.getPlayer().getLocationPlace().getPlaceUpgrades().contains(PlaceUpgrade.LILAYA_MILKING_ROOM)) {
 				MilkingRoom room = Main.game.getOccupancyUtil().getMilkingRoom(Main.game.getPlayerCell().getType(), Main.game.getPlayerCell().getLocation());
 
+				// Fluid amount button listeners:
+				id = "AMOUNT_MINUS_ONE";
+				if (((EventTarget) MainController.document.getElementById(id)) != null) {
+					((EventTarget) MainController.document.getElementById(id)).addEventListener("click", e -> {
+						if (MilkingRoom.INGESTION_AMOUNT >= 6) {
+							MilkingRoom.INGESTION_AMOUNT = MilkingRoom.INGESTION_AMOUNT - 1;
```

Comment:
Doesn't really matter, but you could shorten these lines like this:
```suggestion
							MilkingRoom.INGESTION_AMOUNT -= 1;
```
(and use `+=` for addition)

Reply:
You're right. Done.